hackerank/js-practice/birthday-cake-candles.js

```diff
@@ -1,32 +1,25 @@
-'use strict';
+"use strict";
 // https://www.hackerrank.com/challenges/time-conversion/problem
 function birthdayCakeCandles(candles) {
-    let numMaxCandles = 1;
-    // ciclo for para buscar el numero maximo
-    // IDEA: probar una funcion que se llama max
-    let numeroMasAlto = -1
-    candles.forEach(element => {
-        if (element > numeroMasAlto)
-            numeroMasAlto = element;
-    });
+  // IDEA: probar una funcion que se llama max
+  const numeroMasAlto = Math.max(...candles);
 
-    const result = candles.filter(numCandles => numCandles==numeroMasAlto);
-    return result.length;
+  const result = candles.filter((numCandles) => numCandles == numeroMasAlto);
```

Comment:
@LuciaMezquida , I think it's unnecessary to add new parenthesis because is more readable, you are adding more characters without meaning. 

I recommend removing these `(numCandles)` ->  `numCandles`

Reply:
Ey! Ya lo he corregido!

Reply:
Great! Ship it !!!!